sysdig/resource_sysdig_secure_policy.go

```diff
@@ -335,5 +343,31 @@ func resourceSysdigPolicyUpdate(ctx context.Context, d *schema.ResourceData, met
 	if err != nil {
 		return diag.FromErr(err)
 	}
+	sysdigClients.AddCleanupHook(sendPoliciesToAgents)
+
 	return nil
 }
+
+var sendPoliciesToAgentsOnce sync.Once
+
+func sendPoliciesToAgents(ctx context.Context, clients SysdigClients) error {
```

Comment:
So if there are 10 rule/macro/list changes, how would the behavior be after this change?

Reply:
It will make one call to the API to send the policies to agents.
That's the purpose of using sync.Once.  It guarantees that we call this only once.